### PR TITLE
Add the missing pyjson5 module

### PR DESCRIPTION
**Pull request by tgys**
###### Motivation for this change
 pyjson5 is needed by jupyterlab_server https://github.com/NixOS/nixpkgs/issues/72965

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @FRidh @jonringer

### CODE REVIEW

**Comment by FRidh** [comment]
Please check the contributing guidelines for how to name commits. Merges should be avoided so please rebase instead.

File: pkgs/top-level/all-packages.nix
Comment (veprbl) [inline pkgs/top-level/all-packages.nix]: Things from pkgs/development/python-modules can not go to all-packages.nix, put those in python-packages.nix instead

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: Please add a comment explaining why you have disabled the tests.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: Please don't remove the default configure phase. This is not necessary with `buildPythonPackage`.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: The `buildPythonPackage` implements a better default `buildPhase` that can handle this. So this is not needed, please remove.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: Also not needed.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: So tests work? Then set `doCheck = true;`.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: Most python packages don't need to take `stdenv`, using `lib` should be enough.

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (veprbl) [inline pkgs/development/python-modules/pyjson5/default.nix]: You don't use `python` or `setuptools`.

**Review by veprbl — COMMENTED** [review]
Commit name should read "pythonPackages.pyjson5: init at 0.8.5"

File: pkgs/top-level/python-packages.nix
Comment (veprbl) [inline pkgs/top-level/python-packages.nix]: This line is not needed in python-packages

**Review by jonringer — COMMENTED** [review]
the commit history should look like:

```
maintainers: add isgy
python3Packages.pyjson5: init at 0.8.5
```

**Comment by alexvorobiev** [comment]
Should the derivation for jupyterlab_server be updated as well to add the dependency on pyjson5?

**Comment by jonringer** [comment]
that is what the PR states :)
Personally, i would do that here, but if the PR wants to be broken into 2, then thats also okay

**Comment by isgy** [comment]
@jonringer @veprbl I'm not sure why the tests are failing (since I haven't any of the files in that list)

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Comment by jonringer** [comment]
The error wasn't even from your package, that was weird

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/pyjson5/default.nix]: This is the default for pythonpackages
```suggestion
```

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/pyjson5/default.nix]: personal taste
```suggestion
  meta = with lib; {
```

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/pyjson5/default.nix]:
```suggestion
    license = licenses.asl20;
```

File: pkgs/development/python-modules/pyjson5/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/pyjson5/default.nix]:
```suggestion
    maintainers = with maintainers; [ isgy ];
```

**Comment by FRidh** [comment]
Always rebase, do not merge master.

**Comment by isgy** [comment]
sorry, I know.. fixing

**Comment by isgy** [comment]
@jonringer @FRidh

**Comment by jonringer** [comment]
please merge the two maintainer commits..

```
git rebase -i HEAD~4
# move the bottom (maintainer) line to below the first maintainer entry
# change pick to squash
# save
git push <fork> <branch> --force
```

**Comment by isgy** [comment]
@jonringer @FRidh sorry.. good idea :)

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
commits LGTM

failures are broken on master
```
[6 built (2 failed), 3 copied (9.5 MiB), 9.2 MiB DL]
error: build of '/nix/store/5981hvalbm7s5kgz1sm4wwcd6c5k7pan-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/72974
2 package failed to build:
python37Packages.jupyterlab python38Packages.jupyterlab

5 package were build:
python27Packages.pyjson5 python37Packages.jupyterlab_server python37Packages.pyjson5 python38Packages.jupyterlab_server python38Packages.pyjson5
```

**Pull request merged**
